### DISCUSSION TITLE
Continuous build 139b785  BT ROR display error

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1356] [General]
[23-12-20_2118.alog.txt](https://github.com/artisan-roaster-scope/artisan/files/13728728/23-12-20_2118.alog.txt)
[artisan-settings.aset.txt](https://github.com/artisan-roaster-scope/artisan/files/13728730/artisan-settings.aset.txt)

https://github.com/artisan-roaster-scope/artisan/assets/149906183/250b3b78-e5f6-4756-a62d-e4911654c0e7

**Comment by MAKOMO** [comment]
Thanks for reporting. Should be fixed now. New builds are being generated within the next hour.
